clang/include/clang/Tooling/Syntax/Tokens.h

```diff
@@ -145,6 +145,20 @@ class Token {
 /// For debugging purposes. Equivalent to a call to Token::str().
 llvm::raw_ostream &operator<<(llvm::raw_ostream &OS, const Token &T);
 
+/// A list of tokens as lexed from the input file, without expanding
+/// preprocessor macros.
+class UnexpandedTokenBuffer {
+  std::string Code;
```

Comment:
Do we really need to take a copy of `Code` here?

Reply:
Whoops, copy paste error.